svc/pkg/cluster/src/workflows/server/install/install_scripts/components/traefik.rs

```diff
@@ -34,6 +34,10 @@ pub const TUNNEL_SERVICES: &[TunnelService] = &[
 		name: "vector-tcp-json",
 		port: TUNNEL_VECTOR_TCP_JSON_PORT,
 	},
+	TunnelService {
+		name: "pegboard-server",
+		port: 5030,
```

Comment:
can we make this a const easily

Reply:
It's not used anywhere besides the manager which doesn't import the cluster package